### ISSUE TITLE
Add better `<label>`ing to Address and Name question subfields

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1475]
The sub-input fields that are part of Address and Name questions (ZIP Code, State, City, etc.) don't have associated `<label>`s, so screen readers don't know what the fields are for.

There are indeed `<label>` elements nearby, but they don't use the `for` attribute, so they aren't attributed to the relevant inputs.

![image](https://user-images.githubusercontent.com/77128518/123185383-a670a280-d44a-11eb-8c1f-22d227f0010a.png)

Give the sub-inputs an `id`, and then have the relevant `<label>` use `for="that_id"`. [See this guide](https://css-tricks.com/html-inputs-and-labels-a-love-story/).

## Original tester's report

> Actual results:
At step 6 above, observe the address text form fields are not labelled.
> Expected results:
All controls should have a proper label associated and screen reader should be able to announce the label , state, control type & value as seen applicable

**Comment by ndmckinley** [comment]
This was done in #1492.

**Comment by neel-a-mehta** [comment]
Using a screen reader, I can indeed hear it say things like "Address Line 1", "Address Line 2", etc., including the text I put into the text areas. So this works!
